doc/api/deprecations.md

```diff
@@ -25,10 +26,17 @@ deprecations below. Documentation-only deprecations that support that flag
 are explicitly labeled as such in the
 [list of Deprecated APIs](#list-of-deprecated-apis).
 
-A Runtime deprecation will, by default, generate a process warning that will
-be printed to `stderr` the first time the deprecated API is used. When the
-[`--throw-deprecation`][] command-line flag is used, a Runtime deprecation will
-cause an error to be thrown.
+A Runtime deprecation for only non-`node_modules` code will, by default,
```

Comment:
```suggestion
A runtime deprecation for only non-`node_modules` code will, by default,
```

Reply:
I guess with the suggestion above this should be

```suggestion
An Application deprecation for only non-`node_modules` code will, by default,
```

instead.